aas-web-ui/src/components/SubmodelPlugins/TechnicalData.vue

```diff
@@ -2,17 +2,36 @@
     <v-container fluid class="pa-0">
         <!-- Header -->
         <v-card class="mb-4">
-            <v-card-title>
-                <div class="text-subtitle-1">{{ 'Technical Data:' }}</div>
+            <v-card-title class="pb-0 d-flex justify-space-between align-center">
+                <span>{{ nameToDisplay(submodelElementData) }}</span>
+                <v-switch
+                    v-model="tableView"
+                    color="primary"
+                    label="Table View"
+                    hide-details
+                    density="compact"
+                    class="pa-0"></v-switch>
```

Comment:
That was also my first idea. Unfortunately, the switch then requires extra space within the ‘Technical Properties’ v-expansion-panel-text. I personally didn't find a good-looking placement for the switch here.

Finally I thought it looked quite nice at the top of the v-card-title because it doesn't take up any extra space